cantools/database/can/cpp_source.py

```diff
@@ -226,6 +225,13 @@ def _format_comment_no_tabs(comment):
         return ''
 
 
+# TODO this should evaluate outputs to integers/bools as well, just not only std::string or double
+def _signal_physical_type(signal):
```

Comment:
Dumb question that you've probably already thought about, but does the data structure that `signal` belongs to not have a field that we can run `type()`  function on?

Reply:
Disregard, didn't think this through.